<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1301</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;INVENTORY &lt;span style=&quot; color:#ffaa00;&quot;&gt;BARANG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>16777210</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Transaksi Barang Masuk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Transaksi Barang Keluar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Data Barang</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Data Kategori - Satuan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Data Supplier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Laporan Transaksi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Laporan Stok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: transparent;
    color: #ecf0f1;
    border: none;
    text-align: left;
    font-size: 10pt;

    /* radius hanya kanan */
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;

    margin: 2px 8px;
    padding-left: 12px;
}

/* Hover: strip kiri */
QPushButton:hover {
    background-color: rgba(78, 93, 108, 0.3);
    border-left: 4px solid  rgb(255, 170, 0);
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected */
QPushButton:checked {
    background-color: #2980b9;
    color: white;
    font-weight: bold;
    border-left: 4px solid #ffffff;
}
</string>
          </property>
          <property name="text">
           <string>Management User</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>12</x>
             <y>20</y>
             <width>183</width>
             <height>30</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Selamat Datang,</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelUser">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>49</width>
             <height>30</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);</string>
           </property>
           <property name="text">
            <string>User</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_11">
           <property name="geometry">
            <rect>
             <x>860</x>
             <y>10</y>
             <width>121</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Button sidebar normal */
QPushButton {
    background-color: #ff0000;
    color: #ecf0f1;  /* Putih abu-abu */
    border: none;
    text-align: center;
    font-size: 10pt;
    border-radius: 6px;  /* Radius sedang, tidak terlalu bulat */
    margin: 2px 8px;
}

/* Hover sederhana */
QPushButton:hover {
    background-color: #b81414;  /* Sedikit lebih terang */
    color: white;
}

/* Pressed */
QPushButton:pressed {
    background-color: #2c3e50;
}

/* Active / Selected (pakai checkable=true di Designer) */
QPushButton:checked {
    background-color: #2980b9;  /* Biru sedang */
    color: white;
    font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Keluar</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameContent">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1298</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
